subsys/ipc/ipc_service/backends/ipc_icbmsg.c

```diff
@@ -1249,7 +1246,8 @@ static int backend_init(const struct device *instance)
 	MAYBE_CONST struct icbmsg_config *conf = (struct icbmsg_config *)instance->config;
 	struct backend_data *dev_data = instance->data;
 #ifdef CONFIG_MULTITHREADING
-	static K_THREAD_STACK_DEFINE(ep_bound_work_q_stack, EP_BOUND_WORK_Q_STACK_SIZE);
+	static K_THREAD_STACK_DEFINE(ep_bound_work_q_stack,
+				     CONFIG_IPC_SERVICE_BACKEND_ICBMSG_EP_BOUND_WORK_Q_STACK_SIZE);
```

Comment:
This is just a mild suggestion but we use #define aliases for Kconfig names that are very long. E.g.

https://github.com/nrfconnect/sdk-zephyr/blob/main/drivers/clock_control/clock_control_nrf2_global_hsfll.c#L28

Consider improving it if you do another push to this PR. Otherwise, you can ignore this comment

```
#define EP_BOUND_WORK_Q_STACK_SIZE \
	(CONFIG_IPC_SERVICE_BACKEND_ICBMSG_EP_BOUND_WORK_Q_STACK_SIZE)
```